### ISSUE TITLE
A Sample App would be nice

### ISSUE THREAD

**Issue by antranapp** [immobiliare/RealFlags#20]
### Feature Request

|    Q        |   A
|------------ | ------
| New Feature | yes
| RFC         | no
| BC Break    | no

#### Summary

First of all, thank you for the great library. It looks modern and easy to use.

I'd suggest to add a simple demo app into this library, so that other can easily start play around with the library immediately without initial setup.

**Comment by malcommac** [comment]
Yeah, it would be nice; I can't make any promise to put this in action soon, but I agree with the idea, and I'll schedule it for a future update. Thanks!
